### ISSUE TITLE
Writing Naming section of content style guide

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#2063]
## User Story or Problem Statement
As a content team member, I need to understand the naming conventions used on VA.gov  so I can create and edit copy that adheres to our style guide.

## Goal
Write a section of the style guide to explain VA.gov's naming rules as well as do's and don'ts. 
Draft version can be found here: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/blob/master/src/_content-style-guide/naming.md

## Objectives or Key Results this is meant to further
Ensure that all content written for VA.gov is plain language and user-friendly instead of filled with branded names.

## Acceptance Criteria
- [ ] Naming section written in content style guide, with cross links to other sections of the guide and the Logos section of Design guide.

## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `triage`, `tools-improvements`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `call center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by RLHecht** [comment]
@jenniferlee-dsva I've created a draft for the Naming page. Let me know what you think. 

http://dev-design.va.gov.s3-website-us-gov-west-1.amazonaws.com/content-style-guide/naming

**Comment by jenniferlee-dsva** [comment]
Thank you @RLHecht  . Def on the right direction and you've captured the essential points we want to drive home. Some quick feedback to get you started on v. 2, and then let's tb in convo to finalize. 

1/ Overall, I'd try to make it shorter. Maybe just 1 short paragraph as intro... maybe 2 if we really need it. 

2/ Break up the examples with header sections that convey what we are telling ppl to do or not do specifically. 
- Example - Don't turn plain language words into a brand acronym ??  

3/ It would be nice to recast the 2nd set of examples into an affirmative sentence rather than a "can't" - just a feeling thing. 

4/ I would try to break up the ideas in the paragraph text into sections with clear headers and subheaders - almost like creating sign posts for your main ideas vs. sub-ideas on the page. (I'll paste in a slide from the brown bag on this, in case it helps.)

![image](https://user-images.githubusercontent.com/43381063/66135289-b3d76080-e5c7-11e9-9e80-dd5c8699b7a1.png)


It's looking good! Thank you for capturing these ideas.

**Comment by RLHecht** [comment]
@jenniferlee-dsva I made some revisions. Let me know if you still think it's too long. I think breaking it up visually with heds and the examples makes it easier to read but if it's still too wordy I can cut. 

http://dev-design.va.gov.s3-website-us-gov-west-1.amazonaws.com/content-style-guide/naming

**Comment by jenniferlee-dsva** [comment]
Per Slack - uploading here, too. 
[naming-draft-v2-rh-jl-100819.docx](https://github.com/department-of-veterans-affairs/va.gov-team/files/3704985/naming-draft-v2-rh-jl-100819.docx)

**Comment by Karak888** [comment]
@jenniferlee-dsva @RLHecht going back and forth on content for this section, close to finishing!

**Comment by RLHecht** [comment]
@jenniferlee-dsva I've input all your edits, which all work. I reworked the last bullet that you thought was awkwardly worded. Instead of adding an exceptions section, I added a line about checking the Word List to see what terms are already considered acceptable to capitalize. If you still think it needs the exceptions info, let me know—since we are explaining here that new terms shouldn't become a brand, I didn't think it was necessary to explain what established brands had to do to get there. http://dev-design.va.gov.s3-website-us-gov-west-1.amazonaws.com/content-style-guide/naming

**Comment by jenniferlee-dsva** [comment]
@RLHecht - Thank you for adding a link to the word list - great idea. I fixed the h3's (they should be h2 for proper symantic order), and made some additional edits. Can you review and copy edit, and if they look okay, please publish. 

Also, I think when the Naming page goes live, can you link to this page from the acronyms topic page?  (I wondered if the the branded acronym info should be there, but I think because this page is much more about branding rather than mechanics, it belongs with Naming. However, we should link from that page to this section.)

**Comment by RLHecht** [comment]
This page should go live with the next deployment. Moving to validate and will close issue once done. @Karak888 and @jenniferlee-dsva

**Comment by RLHecht** [comment]
Currently live on site. Closing this issue. FYI @Karak888
